project/Dependencies.scala

```diff
@@ -31,8 +30,8 @@ object Dependencies {
     val akkaHttp2Support = "com.typesafe.akka"            %% "akka-http2-support" % Versions.akkaHttp
     val akkaDiscovery    = "com.typesafe.akka"            %% "akka-discovery"     % Versions.akka
 
-    val scalapbCompilerPlugin = "com.thesamet.scalapb" %% "compilerplugin"  % Versions.scalapb
-    val scalapbRuntime        = "com.thesamet.scalapb" %% "scalapb-runtime" % Versions.scalapb exclude("io.grpc", "grpc-netty")
+    val scalapbCompilerPlugin = "com.thesamet.scalapb" %% "compilerplugin"  % scalapb.compiler.Version.scalapbVersion
+    val scalapbRuntime        = "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion exclude("io.grpc", "grpc-netty")
```

Comment:
It looks like `scalapb-runtime` does not (or no longer?) brings in any `io.grpc` artifacts, so this should not lead to problems with conflicting versions of `io.grpc` on the classpath :+1:.

Reply:
@raboof indeed, that's actually where I initially started the investigation and got very confused as it seems that the version of `scalapb-runtime`, at the time the exclude was added, was https://github.com/scalapb/ScalaPB/blob/v0.6.7/build.sbt, with no obvious dependency to `io.grpc`. I decided to leave it as is, but I guess it would make sense to remove this zombie exclude to ease the future classpath investigations.